apps/electron-app/src/main/services/gmail-service.ts

```
@@ -250,6 +257,33 @@ export class GmailOAuthService {
       view.setBounds(bounds);
     }
 
+    // Configure session permissions specifically for OAuth
+    const session = view.webContents.session;
+
+    // Allow Google OAuth domains to bypass CORS restrictions
+    session.webRequest.onBeforeSendHeaders(
+      { urls: ["https://*.google.com/*", "https://*.googleapis.com/*"] },
+      (details, callback) => {
+        // Remove restrictive headers that might block OAuth
+        delete details.requestHeaders["Origin"];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)